binding-mri/binding-mri.cpp

```diff
@@ -147,7 +147,24 @@ static void mriBindingInit()
 	_rb_define_module_function(mod, "raw_key_states", mkxpRawKeyStates);
 	_rb_define_module_function(mod, "mouse_in_window", mkxpMouseInWindow);
 
+	/* Load global constants */
+	bool has_debug = false;
+	bool has_test = false;
+	bool has_btest = false;
+	for (int i = 1; i < shState->rtData().argc; i++)
+	{
+		const std::string arg = shState->rtData().argv[i];
+		if ("debug" == arg)
```

Comment:
Please use `variable == "constant"` order; we don't need defensive coding when compilers do a good enough job of catching our errors these days.